src/lib.rs

```diff
@@ -21,32 +21,40 @@ assert_eq!(leftpad(s, 7), "  blubb");
 use std::borrow::Cow;
 use std::borrow::Borrow;
 
-/// Pads a string to the given length `len` by inserting the character `pad_char` from the left.
+/// Pads a string to the given number of chars by inserting the character `pad_char` from the left.
 ///
-/// If the given string has a length longer or equal to the desired length, it will be
+/// If the given string has more than or exactly the desired number of codepoints, it will be
 /// returned as-is.
 ///
+/// Codepoints are not graphemes, so the result might always not be what a human would expect.
```

Comment:
I've added an example, but it needs some explanation.
Calculating visual length would require lists of ranges to ignore or count as two.
Filtering everything listed [here](https://en.wikipedia.org/wiki/Combining_character#Unicode_ranges) would be a start, but there are more, like `'ﬃ'` and zero-width chars ([1](https://en.wikipedia.org/wiki/Zero-width_non-joiner), [2](https://en.wikipedia.org/wiki/Zero-width_space))